### ISSUE TITLE
Siamese-structure vs. Classification on retrieval performance

### ISSUE THREAD

**Issue by oyyw** [UKPLab/sentence-transformers#420]
Hi,
I have some FAQs datasets strutured like this:
{
'faq1': [similar sentence1, similar sentence2 ...]
'faqn': ...
}
I did some exps comparing the finetune tasks of siamese-structure(like sentence-Bert) and simple classification both based-on BERT, aiming at improving the retrieval performance; I have several FAQ datasets to train then test on another different FAQ dataset (retrieval);
But I found the model fine-tuned on classification task is better;
Did you have any suggestions? Which loss is better (I tried triplet-loss on my datasets) ?

**Comment by nreimers** [comment]
For classification, you use BERT and you passed two sentences (like [CLS] Question [SEP] Answer) to the network and got a label (similar or not similar)? Or what did you try exactly?

**Comment by gai1995** [comment]
> For classification, you use BERT and you passed two sentences (like [CLS] Question [SEP] Answer) to the network and got a label (similar or not similar)? Or what did you try exactly?

For classification, I just give the sentences in a same FAQ a unique label, and do sentence classification(Just passed one sentence and Label), here I have only questions, similar sentences are also questions with regrad to the standard FAQ;
